taskcluster/ac_taskgraph/routes.py

```diff
@@ -9,9 +9,10 @@
 from taskgraph.transforms.task import index_builder
 
 SIGNING_ROUTE_TEMPLATES = [
-    "index.{trust-domain}.{project}.v1.{variant}.{build_date}.revision.{head_rev}.{component}",
-    "index.{trust-domain}.{project}.v1.{variant}.{build_date}.{component}.latest.",
-    "index.{trust-domain}.{project}.v1.{variant}.{component}.latest",
+    "index.{trust-domain}.{project}.v2.{variant}.{build_date}.revision.{head_rev}.{component}",
```

Comment:
A couple of thoughts:

1. As long as nothing complains that we're bumping the version, I think this is fine.
2. In gecko I see that the `{project}` stays after version, while here is the other way around. I don't have any strong preference, just wanted to call that out. I suppose you wanted to be consistent with the existing routes that we have in this repo?
3. Tiny nit: maybe sort them out the same way they are sorted in gecko so that we can compare more easily next time? 

```
    "index.{trust-domain}.{project}.v2.{variant}.latest.{component}",
    "index.{trust-domain}.{project}.v2.{variant}.{build_date}.revision.{head_rev}.{component}",	
    "index.{trust-domain}.{project}.v2.{variant}.{build_date}.latest.{component}",
    "index.{trust-domain}.{project}.v2.{variant}.revision.{head_rev}.{component}",
```

Reply:
1. I agree. As far as I know, no one is using the v1 routes yet. I haven't publicly tell they exist.
2. Good call. I hadn't see this discrepancy. I'll change it! 
3. Okay. Done!

Reply:
1-2-3. Sounds good